test/auth/authenticate.test.js

```diff
@@ -61,3 +68,55 @@ describe('getAuth', () => {
     expect(mockPublicKeyFunc).toHaveBeenCalledWith(undefined)
   })
 })
+
+describe('checkAuthGroup', () => {
+  const adminGroupId = process.env.ADMIN_AD_GROUP_ID
+
+  it('checkAuthGroup should not throw an error for admins with correct group', () => {
+    const testGroup = 'ADMIN'
```

Comment:
This const is not used. Should this line just be removed?

Reply:
removed